src/simulators/extended_stabilizer/gates.hpp

```diff
@@ -85,6 +86,7 @@ const AER::stringmap_t<Gatetypes> gate_types_ = {
     {"cx", Gatetypes::clifford},   // Controlled-X gate (CNOT)
     {"cz", Gatetypes::clifford},   // Controlled-Z gate
     {"swap", Gatetypes::clifford}, // SWAP gate
+    {"ecr", Gatetypes::clifford},  // ECR Gate
```

Comment:
Hi, I am a junior in the realm of Quantum, Just out of curiosity, Is ECR gate a clifford gate ?